ignite/cmd/scaffold_configs.go

```diff
@@ -0,0 +1,79 @@
+package ignitecmd
+
+import (
+	"strings"
+
+	"github.com/spf13/cobra"
+
+	"github.com/ignite/cli/v28/ignite/pkg/cliui"
+	"github.com/ignite/cli/v28/ignite/pkg/placeholder"
+	"github.com/ignite/cli/v28/ignite/services/scaffolder"
+)
+
+// NewScaffoldConfigs returns the command to scaffold a Cosmos SDK configs into a module.
+func NewScaffoldConfigs() *cobra.Command {
+	c := &cobra.Command{
+		Use:   "configs [configs]...",
+		Short: "Configs for a custom Cosmos SDK module",
+		Long: `Scaffold a new config for a Cosmos SDK module.
+
+A Cosmos SDK module can have configurations. An example of a config is "address prefix" of the
+"auth" module. A config can be scaffolded into a module using the "--module-configs" into
+the scaffold module command or using the "scaffold configs" command. By default 
+configs are of type "string", but you can specify a type for each config. For example:
+
+	ignite scaffold configs foo baz:uint bar:bool
+
+Refer to Cosmos SDK documentation to learn more about modules, dependencies and
+configs.
+`,
+		Args:    cobra.MinimumNArgs(1),
+		PreRunE: migrationPreRunHandler,
+		RunE:    scaffoldConfigsHandler,
+	}
+
+	flagSetPath(c)
+	flagSetClearCache(c)
+
+	c.Flags().AddFlagSet(flagSetYes())
+
+	c.Flags().String(flagModule, "", "module to add the query into. Default: app's main module")
```

Comment:
Something like this would follow the current CLI "default" help format:

```suggestion
	c.Flags().String(flagModule, "", "module to add the query into (default: app's main module)")
```